sqlmesh/core/engine_adapter/base.py

```diff
@@ -214,48 +227,79 @@ def _create_table_from_columns(
             exists: Indicates whether to include the IF NOT EXISTS check.
             kwargs: Optional create table properties.
         """
-        properties = self._create_table_properties(**kwargs)
-        schema: t.Optional[exp.Schema | exp.Table] = exp.to_table(table_name)
+        table = exp.to_table(table_name)
+        primary_key_expression = (
+            [exp.Anonymous(this="PRIMARY KEY", expressions=[exp.to_column(k) for k in primary_key])]
```

Comment:
exp.func("PRIMARY KEY", *exp.to_column(k)...)

but yous houldu se the actual prmiary key


exp.PrimaryKey(expressions=...)